docs/dev-concepts/participant-authentication.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_label: Participant authentication
+sidebar_position: 5
+---
+
+# Participant authentication with XMTP
+
+<!--What do we think about the title of this topic? Consider this definition of "participant" - "Topics have participants, but not users. Client apps and the XMTP network have users, but not participants." Based on this definition I don't think we are authenticating participants, are we? I am thinking of "User authentication" (friendly language) or "Blockchain account authentication" (less friendly, but more accurate?)-->
```

Comment:
Agreed, user is better here. But authentication is also not great, the keys are not just about user authentication, but also message authentication and encryption.

Reply:
Thank you, @mkobetic - you know what - I wonder if the title should be more like "Key generation with XMTP" or "Keys used by XMTP"?

We have the existing "Invitation and message encryption" topic and it should cleanly hook into this Keys topic to explain where encryption gets the keys it needs. 

We have the Signatures topic, which I think is actually about user authentication and likewise, it should cleanly hook into this Keys topic to explain where user authentication gets the keys it needs. 

Once we nail down this Keys topic, I need to go back to the encryption and signatures topics to make sure everything is consistent.

What do you think?